### PR TITLE
Thrift package update test

### PR DESCRIPTION
**Pull request by JochenGuck**
* I have checked that this pull request contains only
  `.travis.yml`/`appveyor.yml` changes or commits merged from the `pkg.template` branch. All other changes send
  to https://github.com/ruslo/hunter. **Yes**

* I have checked that no toolchains removed from CI configs, they are commented
  out instead so other developers can enable them back easily and to simplify
  merge conflict resolution. **Yes**

* I have checked that for every commented out toolchain there is a link to the
  broken CI build page or to the minimum compiler requirements documentation
  so other developers can figure out what was the problem exactly. **Yes**

* I have checked that for every enabled toolchain corresponding package passed
  all stages of update cycle: test/merge/upload/release. **Yes**

### CODE REVIEW

File: cmake/configs/default.cmake
Comment (ruslo) [inline cmake/configs/default.cmake:474]: Doesn't belong here.

> I have checked that this pull request contains only
.travis.yml/appveyor.yml changes or commits merged from the pkg.template branch. All other changes send
to https://github.com/ruslo/hunter. Yes

File: appveyor.yml
Comment (ruslo) [inline appveyor.yml:11]: It's not clear what is the intention. Which toolchain you're trying to add/remove?

**Review by ruslo — CHANGES_REQUESTED** [review]
> This branch cannot be rebased due to conflicts

I can't merge it even technically.

File: cmake/configs/default.cmake
Comment (JochenGuck) [reply cmake/configs/default.cmake:474]: and how should the change from ruslo/hunter be incorporated in the tests?

File: appveyor.yml
Comment (JochenGuck) [reply appveyor.yml:11]: I used the same tool chains as in thrift version 0.11 but updated the reports of the fail of the not supported tool chains

File: cmake/configs/default.cmake
Comment (ruslo) [reply cmake/configs/default.cmake:474]: I will merge it after update https://github.com/ruslo/hunter/pull/1908 will be tested. This branch is only for `.travis.yml`/`appveyor.yml` updates.

File: appveyor.yml
Comment (ruslo) [reply appveyor.yml:11]: > updated the reports of the fail of the not supported tool chains

It was not necessary. The essential part is removing/adding toolchains.

Comment (JochenGuck) [reply appveyor.yml:11]: Ok sorry, than the original appveyor.yml  will also do it

File: cmake/configs/default.cmake
Comment (JochenGuck) [reply cmake/configs/default.cmake:474]: So in the end we can simply skip this pull request because the initial state of this repository has all ready the correct state

**Comment by ruslo** [comment]
> So in the end we can simply skip this pull request because the initial state of this repository has all ready the correct state

> Well, that's precisely the same set of toolchains that are used in pkg.thrift already:

- https://github.com/ruslo/hunter/pull/1908#issuecomment-507590208

**Pull request closed without merging**
